src/Raven.Server/ServerWide/SecretProtection.cs

```diff
@@ -417,6 +429,76 @@ string GetStdError()
             };
         }
 
+        public void NotifyExecutableOfCertificateChange(string executable, string userArgs, string newCertificateBase64, ServerStore serverStore)
+        {
+            Process process;
+
+            var args = $"{userArgs} {CertificateRequestType.CertificateChanged} {newCertificateBase64}";
```

Comment:
Looking at this, I think that this gets too complex for the script.
We might want to simply provide three different exec options

Reply:
Also, not sure if we should pass the cert in the command line. That is actually publicly visible info.
`Get-CimInstance Win32_Process | select CommandLine`

Reply:
I personally would prefer 1 script that can handle all the cases, not X-number of different ones. What is the industry standard for this?

Reply:
I don't think there _is_ such a standard. And the problem with having a single script is that you turn something fairly simple (use certbot) to complex shell script, which I think we should avoid.
A single exec also means versioning issues.

Reply:
How should we pass the cert?

Reply:
Redirect standard input?

Reply:
Standard Input